personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -12,16 +12,47 @@ function PersonalityTestXBlockStudent(runtime, element) {
         return tmp;
     }
 
-    function paragraphize(str) {
+    function paragraphize (str) {
         var tmp = str.split('\n');
```

Comment:
never call variables `tmp`. call it what it is, here it could be `lines`, or `paragraphs`, etc.